docs/setup-cluster/k8s/controller-reviews.rst

```diff
@@ -0,0 +1,173 @@
+.. _controller-reviews:
+
+#############################
+ Gateway API Implementations
+#############################
+
+This document is a survey of the Gateway API controllers that are available and listed by the `SIG
+here <https://gateway-api.sigs.k8s.io/implementations/#haproxy-kubernetes-ingress-controller>`_
+
+*********
+ Contour
+*********
+
+> Contour v1.29.0 implements Gateway API v1.0.0. All Standard channel v1 API group resources
+(GatewayClass, Gateway, HTTPRoute, ReferenceGrant), plus most v1alpha2 API group resources
+(TLSRoute, TCPRoute, GRPCRoute, ReferenceGrant, and BackendTLSPolicy) are supported.
+
+https://projectcontour.io/docs/1.29/guides/gateway-api/
+
+***************
+ Envoy Gateway
+***************
+
+https://gateway.envoyproxy.io/latest/tasks/traffic/tcp-routing/
+
+####################
+ Support Not Tested
+####################
+
+********
+ Cilium
+********
+
+https://docs.cilium.io/en/stable/network/servicemesh/gateway-api/gateway-api/#gs-gateway-api Based
+on Envoy.
```

Comment:
For https:// links use this format

`Link text
<https://url>`_ 

e.g.,

`Cilium Gateway API Support
<https://docs.cilium.io/en/stable/network/servicemesh/gateway-api/gateway-api/#gs-gateway-api>`_